warehouse/ingest-core/src/main/java/datawave/ingest/data/config/ingest/BaseIngestHelper.java

```diff
@@ -306,24 +309,24 @@ public void setup(Configuration config) {
         // Process the reverse index fields
         if (config.get(this.getType().typeName() + DISALLOWLIST_REVERSE_INDEX_FIELDS) != null) {
             if (log.isDebugEnabled()) {
-                log.debug("Disallowlist specified for: " + this.getType().typeName() + DISALLOWLIST_REVERSE_INDEX_FIELDS);
+                log.debug("Disallowlist specified for: {}", this.getType().typeName() + DISALLOWLIST_REVERSE_INDEX_FIELDS);
             }
 
             this.setHasReverseIndexDisallowlist(true);
 
             configProperty = DISALLOWLIST_REVERSE_INDEX_FIELDS;
         } else if (config.get(this.getType().typeName() + REVERSE_INDEX_FIELDS) != null) {
             if (log.isDebugEnabled()) {
-                log.debug("Reverse Index specified.for: " + this.getType().typeName() + REVERSE_INDEX_FIELDS);
+                log.debug("Reverse Index specified.for: {}", this.getType().typeName() + REVERSE_INDEX_FIELDS);
```

Comment:
```suggestion
                log.debug("Reverse Index specified for: {}", this.getType().typeName() + REVERSE_INDEX_FIELDS);
```
Not sure the `.` is needed here.